### PR TITLE
display welcome message

### PR DESCRIPTION
**Pull request by claudiahalip**
Battleship game project built with Spring Boot.
The home page will display a welcome message.

Screen Shot 2021-11-18 at 6 12 50 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,32 @@
-## Battleship with Java
\ No newline at end of file
+## Battleship with Java
+
+ Battleship game built with  Java 11 and  Spring Boot 2.5.6
```
Comment (NicoleCarpenter) [inline README.md:3]: Please clean this README up a bit. You have a couple of places here that have double spaces.

File: README.md [hunk 2]
```diff
@@ -1 +1,32 @@
...
+
+## Installation
+1.Clone the repository to your computer.
```
Comment (NicoleCarpenter) [inline README.md:6]: Add space after `1.` and `2.`

File: README.md [hunk 3]
```diff
@@ -1 +1,32 @@
...
+
+2.Import the app into your IDE through build.gradle file.
+
+## Start
+To start the application run the BattleshipApplication(./src/main/java/com.game.Battleship/BattleshipApplication)
```
Comment (NicoleCarpenter) [inline README.md:11]: Is this supposed to be a command? If so, please put in a code block. If not, please put a space before the `(`

File: README.md [hunk 4]
```diff
@@ -1 +1,32 @@
...
+
+or run in te terminal:
```
Comment (NicoleCarpenter) [inline README.md:13]: Misspelled `the`

File: README.md [hunk 5]
```diff
@@ -1 +1,32 @@
...
+
+``` ./gradlew bootRun```
+
+The server is initialized with port: 8080(http)
+
+For the home page go to:
+
+http://localhost:8080/home 
+
```
Comment (NicoleCarpenter) [inline README.md:22]: Two blank lines here, everywhere else has 1.

File: README.md [hunk 6]
```diff
@@ -1 +1,32 @@
...
+
+## Run tests
+
```
Comment (NicoleCarpenter) [inline README.md:25]: This is the only heading you have a blank space after. Please make the same as the others.

File: README.md [hunk 7]
```diff
@@ -1 +1,32 @@
...
+To start tests, run BattleshipApplicationTests(./src/test/java/com.game.Battleship/BattleshipApplicationTests)
```
Comment (NicoleCarpenter) [inline README.md:26]: Looks like this is a command. Please add in a code block.

File: README.md [hunk 8]
```diff
@@ -1 +1,32 @@
...
+
+or run in the terminal:
+
+```./gradlew test```
+
+
```
Comment (NicoleCarpenter) [inline README.md:32]: Two blank lines at the end not necessary. Try to keep this consistent with all files. On my project, for instance, our linter forces a single blank line at the end of each file.

File: src/main/resources/application.properties [hunk 9]
```diff
@@ -0,0 +1 @@
+
```
Comment (NicoleCarpenter) [inline src/main/resources/application.properties:1]: What is this and why is it empty?

File: src/main/java/com/game/Battleship/BattleshipApplication.java [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class BattleshipApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(BattleshipApplication.class, args);
+	}
+
```
Comment (NicoleCarpenter) [inline src/main/java/com/game/Battleship/BattleshipApplication.java:12]: You have a blank line here but not in src/main/java/com/game/Battleship/controllers/HomeController.java after the `welcome()` method. Please choose whether or not to leave a blank line and be consistent with that.

**Review by NicoleCarpenter — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (claudiahalip) [reply README.md:3]: Done!

File: README.md (hunk 3 above)
Comment (claudiahalip) [reply README.md:11]: It is not a common, it is the path to BattleshipApplication. I've changed a bit the wording.

File: README.md (hunk 4 above)
Comment (claudiahalip) [reply README.md:13]: Done!

File: README.md (hunk 5 above)
Comment (claudiahalip) [reply README.md:22]: Done

File: README.md (hunk 6 above)
Comment (claudiahalip) [reply README.md:25]: Done!

File: README.md (hunk 7 above)
Comment (claudiahalip) [reply README.md:26]: The same as for running the BattleshipApplication, I added more details.

File: src/main/resources/application.properties (hunk 9 above)
Comment (claudiahalip) [reply src/main/resources/application.properties:1]: It was added when I generate the project. I'm not sure what is it for but I will read about it and delete it if it's not necessary.

File: src/main/java/com/game/Battleship/BattleshipApplication.java (hunk 10 above)
Comment (claudiahalip) [reply src/main/java/com/game/Battleship/BattleshipApplication.java:12]: Done!

**Review by NicoleCarpenter — APPROVED** [review]

**Pull request merged**